cmd/wallet-monitor/main.go

```diff
@@ -0,0 +1,109 @@
+package main
+
+import (
+	"math/big"
+	"net/http"
+	"os"
+	"time"
+
+	"contrib.go.opencensus.io/exporter/prometheus"
+	"github.com/urfave/cli/v2"
+	"go.opencensus.io/stats"
+	"go.opencensus.io/stats/view"
+	"go.opencensus.io/tag"
+
+	logging "github.com/ipfs/go-log/v2"
+
+	"github.com/filecoin-project/go-address"
+	"github.com/filecoin-project/lotus/build"
+	"github.com/filecoin-project/lotus/chain/types"
+	cliutil "github.com/filecoin-project/lotus/cli/util"
+)
+
+var (
+	log = logging.Logger("main")
+
+	keyAddress  = tag.MustNewKey("address")
+	balMetric   = stats.Int64("actor/balance", "actor blance", "FIL")
+	nonceMetric = stats.Int64("actor/nonce", "nonce", "n")
+	balanceView = &view.View{
+		Name:        "actor/balance",
+		Measure:     balMetric,
+		Aggregation: view.LastValue(),
+		TagKeys:     []tag.Key{keyAddress},
+	}
+	nonceView = &view.View{
+		Name:        "actor/nonce",
+		Measure:     nonceMetric,
+		Aggregation: view.LastValue(),
+		TagKeys:     []tag.Key{keyAddress},
+	}
+)
+
+func main() {
+	app := &cli.App{
+		Name:        "wallet-monitor",
+		Usage:       "monitor wallet addresses",
+		Version:     build.UserVersion(),
+		Description: "Export wallet balances as metrics",
+		Flags: []cli.Flag{
+			&cli.StringFlag{
+				Name:  "listen",
+				Value: "0.0.0.0:8888",
```

Comment:
Any other port would be good, this will frequently conflict with people developing, any port made of only `8`, and/or `0` is bound to conflict.